### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by zezhong5**
Add tutorial of making and analyzing graph by base r and ggplot2

### CODE REVIEW

File: ggplot_base_r_graph_analysis.Rmd [hunk 1]
```diff
@@ -0,0 +1,115 @@
+---
```
Comment (nicric99) [inline ggplot_base_r_graph_analysis.Rmd:1]: The whole YAML introduction should be cancelled including the --- lines . Instead, fix it with an introduction like this:
#Title of your project
**white line**
Your name
**white line ** 

Put the hashtag in front of the title of your project as it is marked above.

Comment (nicric99) [inline ggplot_base_r_graph_analysis.Rmd:1]: The whole YAML introduction (the part with title: author: etc...) should be cancelled including the --- lines . Instead, fix it with an introduction like this:
#Title of your project
_white line_
Your name
_white line _

Put the hashtag in front of the title of your project as it is marked above.

File: ggplot_base_r_graph_analysis.Rmd [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+date: "10/29/2021"
+output: html_document
+---
+```{r}
+library(ggplot2)
+library(dplyr)
+library(tidyverse)
+library(tidyr)
+```
+***Bar Chart***
```
Comment (nicric99) [inline ggplot_base_r_graph_analysis.Rmd:13]: Suggestion: to create subheadings you can change the '***' to '##'
Remember  just two hashtags, (don't mark subheading with a single #)

**Review by nicric99 — CHANGES_REQUESTED** [review]
The whole YAML introduction (the part with title: author: ... from line 1 to line 6) should be cancelled including the --- lines . Instead, fix it with an introduction like this:
#Title of your project
white line
Your name
_white line _

Put the hashtag in front of the title of your project as it is marked above.


Suggestion: to create subheadings you can change the '***' to '##'
Remember just two hashtags, (don't mark subheading with a single #). For instance the occurrence on lines 13, 27, etc..

**Comment by jtr13** [comment]
@nicric99 It looks like all the changes that were requested were made a week ago. Can you please review and merge if everything looks ok? Thanks.

**Review by nicric99 — CHANGES_REQUESTED** [review]
Other hashtag headers should not be followed by numbers since the hashtags will create numbered subheadings. Correct: ## Subheading. Incorrect: ## 3. Subheading.

**Comment by nicric99** [comment]
Congrats 🎉

**Pull request merged**